3rd-party/CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ option(YAML_CPP_BUILD_CONTRIB OFF)
 option(YAML_CPP_BUILD_TESTS OFF)
 
 add_subdirectory(yaml-cpp EXCLUDE_FROM_ALL)
+set_target_properties(yaml-cpp PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
Why do we need PIC here? Anyway, I'm OK with it, especially because it's third party. Just asking out of curiosity.

Reply:
Without it, I was simply getting a compilation error that said it needs PIC to compile some functions in that library :)